govtool/frontend/src/context/wallet.tsx

```diff
@@ -657,8 +657,10 @@ const CardanoProvider = (props: Props) => {
           targetDRep = DRep.new_always_abstain();
         } else if (target === AutomatedVotingOptionDelegationId.no_confidence) {
           targetDRep = DRep.new_always_no_confidence();
-        } else if (target.includes("drep")) {
+        } else if (target.includes("drep1")) {
           targetDRep = DRep.new_key_hash(Ed25519KeyHash.from_bech32(target));
+        } else if (target.includes("drep_script1")) {
+          targetDRep = DRep.new_script_hash(Ed25519KeyHash.from_bech32(target));
```

Comment:
target should be:
```suggestion
          targetDRep = DRep.new_script_hash(ScriptHash.from_hex(target));
```

where `target` is the `scriptHash` of the script based DRep that is missing in the SQL.

I'll make the reference implementation of this and share you soon